backend/doc/apache.md

```diff
@@ -30,23 +30,36 @@ Create a file named `/etc/apache2/sites-available/tracim.conf` containing:
             Dav On
         </Directory>
         SetEnv proxy-sendcl
-        ProxyPreserveHost On
 
-        ProxyPass /webdav http://127.0.0.1:3030/webdav
-        ProxyPassReverse /webdav http://127.0.0.1:3030/webdav
+        # Setting Destination header from https to http in proxy
+        # is needed for working move/copy in webdav
+        RequestHeader edit Destination ^https http early
```

Comment:
We should consider move this to /webdav location as this is the only known place where we do need this.

Reply:
Done.